### DISCUSSION TITLE
Show scan percentage on `-v`

### DISCUSSION THREAD

**Discussion by rew1nter** [projectdiscovery/naabu#948] [General]
### Please describe your feature request:

It'd be great if you could show the scan percentage during verbose output. What's better is if you can manage to update the percentage on enter like nmap 

### Describe the use case of this feature:

show how the scan progresses. helpful for large scans

**Comment by Mzack9999** [comment]
Please check if this might help: https://github.com/projectdiscovery/naabu/discussions/947 (the scan percentage can be calculated as `packets / total`)

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
